content/pt-br/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -136,16 +138,28 @@ Para mais detalhes sobre compatibilidade entre as versões, veja:
    sudo apt-get install -y apt-transport-https ca-certificates curl gpg
    ```
 
-2. Faça o download da chave de assinatura pública da Google Cloud:
+2. Faça o download da chave de assinatura pública de pacotes do Kubernetes:
 
    ```shell
-   sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+   curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
+`{{< param "version" >}}` é a versão do Kubernetes que você deseja instalar. Por exemplo, `1.30`.
+
+Se o diretório `/etc/apt/keyrings` não existir, você deve criar o diretório:
+
+```shell
+sudo mkdir -p -m 755 /etc/apt/keyrings
+```
+
+{{< nota >}}
+Em versões anteriores ao Debian 12 e Ubuntu 22.04, o diretório `/etc/apt/keyrings` não existe por padrão e deve ser criado antes do comando curl.
+{{< /nota >}}
```

Comment:
The PR build failed [(logs here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6629625e876f0b0009c1455e) due an unknown shortcode named "nota". To address this, please ensure not to localize the shortcode name.

```suggestion
{{< note >}}
Em versões anteriores ao Debian 12 e Ubuntu 22.04, o diretório `/etc/apt/keyrings` não existe por padrão e deve ser criado antes do comando curl.
{{< /note >}}
```

Reply:
I removed the {{< note >}} block.